test/test_helper.rb

```diff
@@ -4,7 +4,9 @@
 require 'vcr'
 
 VCR.configure do |c|
-  c.filter_sensitive_data("<TYPEFORM_API_KEY>") { Typeform::Typeform.api_key }
   c.cassette_library_dir = "test/fixtures"
   c.hook_into :webmock
+  c.before_record do |i|
+    i.response.body.force_encoding('UTF-8')
```

Comment:
can we use here `try`? `i.response.body.try(: force_encoding, 'UTF-8')`

Reply:
Or `&.`?

Reply:
yes. But it's just helper. maybe we can ignore this.

Reply:
I think is no necessary, HTTP responses always have a body

Reply:
ok